avocado/utils/asset.py

```diff
@@ -477,6 +478,26 @@ def get_asset_by_name(cls, name, cache_dirs, expire=None, asset_hash=None):
 
         raise OSError("File %s not found in the cache." % name)
 
+    @classmethod
+    def get_assets_by_atime(cls, days, cache_dirs):
```

Comment:
Naming suggestion: `get_assets_unusued_for_days()`.  IMO the atime usage is an implementation detail.